src/apurva/PrimeNumber.java

```diff
@@ -0,0 +1,30 @@
+package apurva;
+
+import java.util.Scanner;
+
+//Print all prime numbers between 2 to 100.
+public class PrimeNumber {
+	void printPrime(int num) {
+		String primeNumbers = "";
+		for (int index = 2; index <= num; index++) {
```

Comment:
requirement is user will two limit, but looks like you give only one limit which is upper.
No change require just check requirement.